demo/Makefile

```diff
@@ -0,0 +1,80 @@
+
+# Domain to use when starting flynn
+CLUSTER_DOMAIN := demo.localflynn.com
+CLUSTER := default
+BOOTSTRAP_LOG_FILE := bootstrap.${CLUSTER}.log
+
+# Providers: virtualbox, vmware_fusion, aws
+PROVIDER := virtualbox
+
+
+.PHONY: install-cli init bootstrap up down halt update ssh reset token destroy dashboard ssh-cmd
+
+
+# Install flynn cli on host
+install-cli:
+	L=/usr/local/bin/flynn && curl -sL -A "`uname -sp`" https://dl.flynn.io/cli | zcat >$L && chmod +x $L
+
+# Initialize a new cluster
+init:
+	@$(MAKE) up
+	@$(MAKE) bootstrap
+	flynn key add
+	@printf "\nFlynn is now running in the VM.\nOpen dashboard: make dashboard\n\n"
+
+# Bootstrap the cluster
+# This should only be called once per cluster
+# Runs bootstrap in VM and cluster add on host
+bootstrap:
+	@$(MAKE) CMD='CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" flynn-host bootstrap /etc/flynn/bootstrap-manifest.json 2>&1' ssh-cmd | tee ${BOOTSTRAP_LOG_FILE}
+	@(grep '^flynn cluster add .*$$' ${BOOTSTRAP_LOG_FILE} || echo exit 1) | sed "s/$$(printf '\r')\$$//" | bash
```

Comment:
Best bet is to rewrite the `cluster add` command since we need to add the force/default flags anyway. That message is only read by people using the manual install script, all of the orchestration tools use the JSON output from bootstrap and build the command using data from the relevant steps.